huangjoyce3/MealMatch
# Assign Volunteers backend

Implemented backend for volunteer assignment flow.

**Changes made:**
- query deliveries assigned to deliverer group
- render deliveries and information
- allow assigning of volunteers and modifying of assigned volunteers
- autoformat telephone number as it's typed (just for funsies)

**Testing:**
- Navigate to the deliveries page
- Choose a delivery that has no volunteers assigned to it
- assign two volunteers
- confirm that the database shows the two new volunteers
- choose the same delivery or another delivery that already has volunteers
- modify those volunteers
- confirm that those updates appear in the db

**TODOS:**
- email/link after confirmation?
- Replace states in `AssignVolunteersController` with state Enum
- routing?

**Questions:**
- can we do the DA and RA metadata queries all at once?
- do we have to have 2 volunteers?
